openshift/mlflow-experiment-image-buildconfig.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-experiment-image-build-template
+
+parameters:
+- description: Application name
+  value: 'mlflow-experiment-image'
+  name: APPLICATION_NAME
+  required: true
+- name: GIT_URI
+  value: https://github.com/4n4nd/experiment-tracking-template.git
```

Comment:
default URL should point to https://github.com/AICoE/experiment-tracking-template.git